ravens/tasks/tasks_test.py

```diff
@@ -21,67 +21,132 @@
 from ravens.environments import environment
```

Comment:
Please revert the formatting changes in this file (move back to 2 space indent and single quotes for strings)